### PR TITLE
add new extension - sticky header

### PR DESCRIPTION
**Pull request by vinzloh**
An extension which provides a sticky header for table columns when scrolling on a long page and / or table. Works for tables with many columns and narrow width with horizontal scrollbars too

### CODE REVIEW

**Comment by wenzhixin** [comment]
1. Can you help to add an example in: https://github.com/wenzhixin/bootstrap-table-examples/tree/master/extensions repo?
2. Please add an `extension.json` file just like [other extension](https://github.com/wenzhixin/bootstrap-table/blob/develop/src/extensions/editable/extension.json), because our extension docs is based on this file.
3. You can use [sprintf](https://github.com/wenzhixin/bootstrap-table/blob/master/src/extensions/filter-control/bootstrap-table-filter-control.js#L11) method in your code, for example like this: `sprintf('<div id="%s" class="hidden"></div>', sticky_header_container_id)`, etc...

File: src/extensions/sticky-header/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Table sticky-header
+
+This is an extension for [Bootstrap table](http://github.com/wenzhixin/bootstrap-table) module which provides a sticky header for the table when scrolling. </br>
+You must include the bootstrap-table-sticky-header.css file in order to get the appropriate style
+
+## Usage
+
+```html
+<script src="extensions/group-by/bootstrap-table-sticky-header.js"></script>
```
Comment (djhvscf) [inline src/extensions/sticky-header/README.md]: Please fix the path

Comment (vinzloh) [inline src/extensions/sticky-header/README.md]: Hi @wenzhixin,

This is my first pull request.

For adding an example, I can't git push to master branch in: https://github.com/wenzhixin/bootstrap-table-examples/tree/master/extensions repo

I've committed to my local repo

**Comment by wenzhixin** [comment]
Yes, you can create pull request to `bootstrap-table-examples` just like this.

**Pull request merged**